neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -240,6 +240,32 @@ def test_wallet_create_address(self):
         self.assertEqual(type(res), UserWallet)
         self.assertEqual(len(res.Addresses), 9)
 
+    def test_wallet_delete_address(self):
+        # test wallet delete address with no wallet open
+        args = ['delete_addr']
+        res = CommandWallet().execute(args)
+        self.assertFalse(res)
+
+        self.OpenWallet()
+
+        # test wallet delete address with no argument
+        args = ['delete_addr']
+        res = CommandWallet().execute(args)
+        self.assertFalse(res)
+
+        # test wallet delete address with unknown address
+        args = ['delete_addr', self.watch_addr_str]
+        res = CommandWallet().execute(args)
+        self.assertTrue(res)  # Current specs.
```

Comment:
So as you noticed `DeleteAddress` _always_ returns `True`. Which can give confusing user feedback like the following for an address that's not actually in the wallet:
```
neo> wallet delete_addr AJN2SZJuF7j4mvKaMYAY9N8KsyD4j1fNdf
Deleted address AJN2SZJuF7j4mvKaMYAY9N8KsyD4j1fNdf
```

It's not really related to the migration but how the underlying `DeleteAdress` works. Would you like to fix it or shall I make it a separate issue?

Reply:
It's a small fix so I'll do it in this PR!